setup.py

```diff
@@ -41,10 +41,10 @@
     description="Unofficial Linux GUI client for ProtonVPN",
```

Comment:
Missed a spot. Btw should we add the readme to the pypi docs or just re-route to Github ?

Reply:
It would be nice if the Readme appears on Pypi indeed.

Reply:
Can I leave the adding of README to setup.py to you? I am no-longer fresh on setup.py :-)

Reply:
@kaplun It is already added. If you check the pypi page for the project, you'll see the readme. I was just making sure if we should have it or not.